src/frontend/components/DashboardVideoLive/OnStageRequestToast/index.tsx

```diff
@@ -0,0 +1,59 @@
+import {
+  useLivePanelState,
+  LivePanelItem,
+} from 'data/stores/useLivePanelState';
+import { Box, Button, Text } from 'grommet';
+import React from 'react';
+import toast from 'react-hot-toast';
+import { defineMessages, useIntl } from 'react-intl';
+
+import { Participant } from 'types/Participant';
+import { generateOnStageRequestNotificationMessage } from 'utils/generateOnStageRequestNotificationMessage';
```

Comment:
I think this name is a bit long, what about 
```suggestion
import { onStageRequestMessage } from 'utils/onStageRequestMessage';
```

Reply:
done